lib/modules/keyboardNav.js

```diff
@@ -1006,28 +1001,24 @@ addModule('keyboardNav', (module, moduleID) => {
 		const selected = modules['selectedEntry'].selected();
 		if (!selected) return;
 
-		if (selected.thing.classList.contains('link')) {
-			// Ahh, we're not in a comment, but in the main story... that key should follow the link.
-			module.followLink();
-			return;
-		} else {
-			// find out if this is a collapsed or uncollapsed view...
-			let thisToggle = selected.entry.querySelector('a.expand');
+		if (selected.thing.classList.contains('link')) return;
```

Comment:
This is really the only line that was changed in this function, the rest was reindented